packages/worker/src/transcribe.ts

```diff
@@ -127,7 +127,7 @@ export const convertToWav = async (
 		};
 	} catch (error) {
 		console.log('ffmpeg failed error:', error);
-		throw error;
+		return;
```

Comment:
what's the motivation for returning void here vs undefined. Is it because there are other failure states that return void? I'm not bothered, but I think elsewhere we have tended towards returning undefined in failure situtations. It is a shame typescript doesn't have a built in Either type!

Reply:
I'd always used void and undefined as return types interchangeably but [the top answer to this question](https://stackoverflow.com/questions/58885485/why-does-typescript-have-both-void-and-undefined) is a good explanation of the distinction. I'll change to undefined since that's what we're doing elsewhere.